### PR TITLE
simplify foreach source code comment

### PR DESCRIPTION
**Pull request by wilzbach**
- just for fun is a bad motivation
- found a missing comma in the text

### CODE REVIEW

File: public/content/en/basics.md [hunk 1]
```diff
@@ -857,11 +857,7 @@ void main() {
           [2, 3, 2, 3], // 10
           [3, 6, 2, 9] ]; // 20
 
-    // This is just for the fun of it: iterate
-    // through the testers in reverse order. This
-    // integrates very nice with foreach because
-    // you just apply another algorith before
-    // you really iterate over the real object.
+    // We iterate through the testers array in reverse order
```
Comment (stonemaster) [inline public/content/en/basics.md]: 48 chars

**Comment by wilzbach** [comment]
rebased - i also realized that `tester` is a horrible name and changed it to `arr`.

File: public/content/en/basics.md [hunk 2]
```diff
@@ -853,24 +853,20 @@ large types. To prevent copying or enable *in-place
...
+    foreach (row; retro(arr))
     {
         double accumulator = 0.0;
-        foreach (c; tester)
+        foreach (c; row)
             accumulator += c;
 
-        writeln("The average of ", tester,
-            " = ", accumulator / tester.length);
+        writeln("The average of ", arr,
```
Comment (stonemaster) [inline public/content/en/basics.md]: This sould be row here

File: public/content/en/basics.md [hunk 3]
```diff
@@ -853,24 +853,20 @@ large types. To prevent copying or enable *in-place
...
+            " = ", accumulator / arr.length);
```
Comment (stonemaster) [inline public/content/en/basics.md]: Row

**Comment by wilzbach** [comment]
Sorry - rebased.

**Pull request merged**